docs/store/README.md

```diff
@@ -67,7 +69,7 @@ import { StoreModule } from '@ngrx/store';
 import { counterReducer } from './counter';
 
 @NgModule({
-  imports: [BrowserModule, StoreModule.forRoot({ count: counterReducer })],
+  imports: [StoreModule.forRoot({ count: counterReducer })],
```

Comment:
Why did you remove `BrowserModule`?

Reply:
It is not imported, and it is not required to understand the example.

We can include the full `AppModule` which complements the rest of the code snippets in this document, but I prefer to keep it simple.

Here's how the full `AppModule` looks like:

```ts
import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';
import { StoreModule } from '@ngrx/store';
import { AppComponent } from './app.component';
import { counterReducer } from './counter';
import { MyCounterComponent } from './my-counter.component';

@NgModule({
  declarations: [
    AppComponent, MyCounterComponent
  ],
  imports: [
    BrowserModule,
    StoreModule.forRoot({ count: counterReducer })
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
```